cabal-install/Distribution/Client/ProjectBuilding.hs

```diff
@@ -612,7 +663,8 @@ rebuildTargets verbosity
                           installPlan pkgsBuildStatus $ \downloadMap ->
 
       -- For each package in the plan, in dependency order, but in parallel...
-      InstallPlan.execute jobControl keepGoing (DependentFailed . packageId)
+      InstallPlan.execute jobControl keepGoing
+                          (BuildFailure Nothing . DependentFailed . packageId)
```

Comment:
Probably is more forward looking to have a `mkBuildFailureNoLog` function instead.